<ui version="4.0" >
 <class>CalendarToolWidget</class>
 <widget class="QWidget" name="CalendarToolWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Calendar Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QPushButton" name="m_insertButton" >
     <property name="text" >
      <string>Insert</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Start date:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="QToolButton" name="m_selectStartDateButton" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="KDateWidget" name="m_startDateWidget" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>End date:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" >
    <widget class="QToolButton" name="m_selectEndDateButton" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="KDateWidget" name="m_endDateWidget" />
   </item>
   <item row="5" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateWidget</class>
   <extends>QWidget</extends>
   <header>kdatewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
